### PR TITLE
[bitnami/keycloak] feat: add support for `KEYCLOAK_LOG_OUTPUT`

### PR DESCRIPTION
**Pull request by marrws**
### Description of the change

This PR allows users to set `KEYCLOAK_LOG_OUTPUT` to configure the `log-console-output` setting. At the moment, the only 2 formats allowed by keycloak are `default` and `json` [^1]

### Benefits

Users can set `logging.output` in the `values.yml` file to configure their desired log output format.

### Applicable issues

Support for this env-var was added in  bitnami/bitnami-docker-keycloak#83

### Additional information

- [Configuring JSON or plain console logging - Keycloak Documentation](https://www.keycloak.org/server/logging#_configuring_json_or_plain_console_logging)
- All configuration - Keycloak Documentation [^1]

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

[^1]: https://www.keycloak.org/server/all-config#_logging

### CODE REVIEW

File: bitnami/keycloak/values.yaml [hunk 1]
```diff
@@ -949,3 +949,12 @@ externalDatabase:
...
 cache:
   enabled: false
+
+## @section Keycloak Logging parameters
+
+## Keycloak logging configuration
+## ref: https://www.keycloak.org/server/logging
+## @param logging.output Alternates between the default log output format or json format
+##
+logging:
```
Comment (javsalgar) [inline bitnami/keycloak/values.yaml:959]: Does it make sense to make it a nested parameter if only there's going to be one value?

Comment (javsalgar) [reply bitnami/keycloak/values.yaml:959]: I'd rather call it loggingOutput and not create a section for it

Comment (marrws) [reply bitnami/keycloak/values.yaml:959]: My reasoning was futureproofing it. Other users may want to add support for `log-console-format` or `log-file-format` inside the same key.

Example:
```yaml
logging:
  output: "default"
  formatFile: "%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"
  formatConsole: "%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"
```

Comment (javsalgar) [reply bitnami/keycloak/values.yaml:959]: Fair enough

**Review by javsalgar — APPROVED** [review]

**Pull request merged**